.github/workflows/deploy.yml

```
@@ -251,21 +272,33 @@ jobs:
+          name = "vibechecc-pr-${{ inputs.pr_number }}-frontend"
+          EOF
+
       - name: Deploy to Cloudflare Workers
         uses: cloudflare/wrangler-action@v3
         with:
           apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
-          command: 'deploy --env ${{ inputs.environment }} --config apps/frontend/browser/wrangler.toml'
+          command: |
+            bunx wrangler deploy --env ${{ inputs.environment == 'ephemeral' && format('pr-{0}', inputs.pr_number) || inputs.environment }} --config apps/frontend/browser/wrangler.toml
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)